Update Crash dump doc (#39456)
dotnet/docs: docs/core/testing/unit-testing-mstest-runner-extensions.md

| `⁠-⁠-⁠crashdump-⁠filename` | Specifies the file name of the dump. |
| `--crashdump-type` | Specifies the type of the dump. Valid values are `Mini`, `Heap`, `Triage`, `Full`. Defaults as `Full`. For more information, see [Types of mini dumps](../diagnostics/collect-dumps-crash.md#types-of-mini-dumps). |

> [!CAUTION]

> > The extension is not compatible with .NET Framework. For .NET Framework you can enable the postmortem debugging with Sysinternals ProdDump as explained in the [documentation](https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/enabling-postmortem-debugging#window-sysinternals-procdump).

Reviewer: ```suggestion
> The extension is not compatible with .NET Framework. For .NET Framework you can enable the postmortem debugging with Sysinternals ProdDump. Follow [this article](https://learn.microsoft.com/windows-hardware/drivers/debugger/enabling-postmortem-debugging#window-sysinternals-procdump) for full guidance.
```

> > The extension is not compatible with .NET Framework and will be silently ignored. For .NET Framework you can enable the postmortem debugging with Sysinternals ProdDump (follow [this article](https://learn.microsoft.com/windows-hardware/drivers/debugger/enabling-postmortem-debugging#window-sysinternals-procdump) for full guidance). The postmortem debugging solution will also collect process crash for .NET so you can avoid the use of the extension if you've mixed .NET/.NET Framework test applications.

Reviewer 2: ```suggestion
> The extension isn't compatible with .NET Framework and will be silently ignored. For .NET Framework support, you enable the postmortem debugging with Sysinternals ProcDump. For more information, see [Enabling Postmortem Debugging: Window Sysinternals ProcDump](windows-hardware/drivers/debugger/enabling-postmortem-debugging#window-sysinternals-procdump). The postmortem debugging solution will also collect process crash information for .NET so you can avoid the use of the extension if you're targeting both .NET and .NET Framework test applications.
```

> > The extension isn't compatible with .NET Framework and will be silently ignored. For .NET Framework support, you enable the postmortem debugging with Sysinternals ProcDump. For more information, see [Enabling Postmortem Debugging: Window Sysinternals ProcDump](windows-hardware/drivers/debugger/enabling-postmortem-debugging#window-sysinternals-procdump). The postmortem debugging solution will also collect process crash information for .NET so you can avoid the use of the extension if you're targeting both .NET and .NET Framework test applications.

Reviewer 2: ```suggestion
> The extension isn't compatible with .NET Framework and will be silently ignored. For .NET Framework support, you enable the postmortem debugging with Sysinternals ProcDump. For more information, see [Enabling Postmortem Debugging: Window Sysinternals ProcDump](/windows-hardware/drivers/debugger/enabling-postmortem-debugging#window-sysinternals-procdump). The postmortem debugging solution will also collect process crash information for .NET so you can avoid the use of the extension if you're targeting both .NET and .NET Framework test applications.
```

> > The extension is not compatible with .NET Framework. For .NET Framework you can enable the postmortem debugging with Sysinternals ProdDump. Follow [this](https://learn.microsoft.com/windows-hardware/drivers/debugger/enabling-postmortem-debugging#window-sysinternals-procdump) for full guidance. In this case the dump will be collected also in case of process crash for .NET and you can avoid to use the extension if you've mixed .NET/.NET Framework test applications. The usage of the extension with .NET Framework applications will be silently ignored.

Reviewer: ```suggestion
> The extension is not compatible with .NET Framework and will be silently ignored. For .NET Framework you can enable the postmortem debugging with Sysinternals ProdDump (follow [this article](https://learn.microsoft.com/windows-hardware/drivers/debugger/enabling-postmortem-debugging#window-sysinternals-procdump) for full guidance). The postmortem debugging solution will also collect process crash for .NET so you can avoid the use of the extension if you've mixed .NET/.NET Framework test applications.
```
